project-base/src/Shopsys/ShopBundle/Resources/scripts/styleguide/plugins/codemirror/codemirror.minified.js

```diff
@@ -0,0 +1 @@
& (h.top < p.top - 2 ? (N(h.right, h.top, null, h.bottom), N(L, p.top, p.left, p.bottom)) : N(h.right, h.top, p.left - h.right, h.bottom)), h.bottom < p.top && N(L, h.bottom, null, p.top); }t.appendChild(i); } function kn (e) { if (e.state.focused) { var t = e.display; clearInterval(t.blinker); var r = !0; t.cursorDiv.style.visibility = '', e.options.cursorBlinkRate > 0 ? t.blinker = setInterval(function () { return t.cursorDiv.style.visibility = (r = !r) ? '' : 'hidden'; }, e.options.cursorBlinkRate) : e.options.cursorBlinkRate < 0 && (t.cursorDiv.style.visibility = 'hidden'); } } function Cn (e) { e.state.focused || (e.display.input.focus(), Tn(e)); } function Sn (e) { e.state.delayingBlurEvent = !0, setTimeout(function () { e.state.delayingBlurEvent && (e.state.delayingBlurEvent = !1, Ln(e)); }, 100); } function Tn (e, t) { e.state.delayingBlurEvent && (e.state.delayingBlurEvent = !1), e.options.readOnly != 'nocursor' && (e.state.focused || (ft(e, 'focus', e, t), e.state.focused = !0, D(e.display.wrapper, 'CodeMirror-focused'), e.curOp || e.display.selForContextMenu == e.doc.sel || (e.display.input.reset(), b && setTimeout(function () { return e.display.input.reset(!0); }, 20)), e.display.input.receivedFocus()), kn(e)); } function Ln (e, t) { e.state.delayingBlurEvent || (e.state.focused && (ft(e, 'blur', e, t), e.state.focused = !1, L(e.display.wrapper, 'CodeMirror-focused')), clearInterval(e.display.blinker), setTimeout(function () { e.state.focused || (e.display.shift = !1); }, 150)); } function Mn (e) { for (var t = e.display, r = t.lineDiv.offsetTop, n = 0; n < t.view.length; n++) { var i = t.view[n]; var o = void 0; if (!i.hidden) { if (x && k < 8) { var a = i.node.offsetTop + i.node.offsetHeight; o = a - r, r = a; } else { var l = i.node.getBoundingClientRect(); o = l.bottom - l.top; } var s = i.line.height - o; if (o < 2 && (o = cn(t)), (s > 0.005 || s < -0.005) && (de(i.line, o), An(i.line), i.rest)) for (var c = 0; c < i.rest.length; c++)An(i.rest[c]); } } } function An (e) { if (e.widgets) for (var t = 0; t < e.widgets.length; ++t) { var r = e.widgets[t]; var n = r.node.parentNode; n && (r.height = n.offsetHeight); } } function Nn (e, t, r) { var n = r && r.top != null ? Math.max(0, r.top) : e.scroller.scrollTop; n = Math.floor(n - Or(e)); var i = r && r.bottom != null ? r.bottom : n + e.wrapper.clientHeight; var o = he(t, n); var a = he(t, i); if (r && r.ensure) { var l = r.ensure.from.line; var s = r.ensure.to.line; l < o ? a = he(t, _e(se(t, o = l)) + e.wrapper.clientHeight) : Math.min(s, t.lastLine()) >= a && (o = he(t, _e(se(t, s)) - e.wrapper.clientHeight), a = s); } return { from: o, to: Math.max(a, o + 1) }; } function On (e) { var t = e.display; var r = t.view; if (t.alignWidgets || t.gutters.firstChild && e.options.fixedGutter) { for (var n = fn(t) - t.scroller.scrollLeft + e.doc.scrollLeft, i = t.gutters.offsetWidth, o = n + 'px', a = 0; a < r.length; a++) if (!r[a].hidden) { e.options.fixedGutter && (r[a].gutter && (r[a]
```

Comment:
This is so, so crazy. Call glory to npm :smile:

Reply:
yep... it is :smile: